### DISCUSSION TITLE
Manually Building from Source - No Man Pages

### DISCUSSION THREAD

**Discussion by BenAigan** [git-lfs/git-lfs#4499] [Q&A] [answered]
I have managed to compile git-lfs from source however I would like to also have the man pages included for my users, is there a way to do this?

**Comment by bk2204** [comment, marked as answer]
Yes, you can do this by installing ronn and running `make man`.  The `ronn` package is available on Debian-based distributions and can be installed on Red Hat-based systems with `gem install ronn`.

**Comment by BenAigan** [comment]
Thanks for the info and the quick response, I have been able to generate the man html pages successfully on a few machines however I am not sure how to "join" up the help to the usage text of git-lfs e.g.

# git lfs
git-lfs/2.13.0 (GitHub; linux arm; go 1.11.6)
Sorry, no usage text found for "git-lfs"

As for the man pages, is there instructions I missed on how to deploy these into the man area?

**Reply by bk2204** [reply]
You're better off building with `make`, which invokes `go generate` to add the help text.  That results in the output being basked into the binaries if it isn't already.

As for installing the manual pages, usually those go into `/usr/share/man` (for distro packages) or `/usr/local/share/man` (for non-distro packages).  However, the presence or absence of those manual pages doesn't affect what's used by the `git-lfs` binary.
